projects/DynamicAttributeRouting/README.md

```diff
@@ -0,0 +1,76 @@
+
+## Project Components
+* CloudFormation template that builds:
+  * IAM Role: Allows Lambda read/write access to Dynamo DB
+  * Lambda Functions
+    * Retrieve dynamic attributes from various tables
+    * Init function to populate DynamoDB tables
+  * DynamoDB Tables
+    * Stores dynamic attributes for lambda functions, dialed numbers and prompts
+* Amazon Connect Contact Flows
+  * Init flow that sets customer experience and retrieves attributes from DynamoDB
+  * Language flow that sets the correct voice and plays language specific introductory prompts
+  * Main flow that sets the dynamic queue, performs hours/staffing checks and places the customer into queue
+  
+## Project Requirements
+* Operational Amazon Connect instance
+* Three (3) claimed inbound numbers
+  * You will claim one inbound number for each language
+  ![dialed_numbers-1](Images/dialed_numbers-1.png)
+* Three (3) unique queues
+  * Make note of the ARN for each queue as you will need them when deploying the CloudFormation stack
+  ![queue_arn](Images/queue_arn.png)
+  * Associate all the queues to an existing routing profile
+  ![routing_profile](Images/routing_profile.png)
+
+## Deployment Steps
+Perform the following steps to deploy this project: 
+1. Create a new CloudFormation stack and upload template `dynamic-attribute-routing.yaml`
+  * Make sure that you select the correct region to match your Amazon Connect instance
+2. Name your stack and enter all the parameters listed with details from your Amazon Connect instance
+  * Take note of the `DeploymentEnvironment` you select
+  * Make sure you enter enter the correct phone number mapping and queue ARNs you noted previously
+  ![cf_parameters](Images/cf_parameters.png)
+3. Click `Next`, select the IAM resource disclaimer and click `Create Stack`
+4. Once the stack creation has completed, add the `~GetLambdas`, `~GetDynAttributes`, and `~GetPrompts` Lambda functions to your Amazon Connect instance.
+  ![associate_lambdas](Images/associate_lambdas.png)
+5. Download the three contact flows in the ContactFlows directory.
+6. Import them in the order listed below. Save and Publish each contact flow after import while making the edits as indicated below:
+  * (MAIN FLOW) `100_MainFlow` - Publish as-is
+  * (LANGUAGE FLOW) `100_LangFlow` - Publish as-is
+  * (INIT FLOW) `100_InitFlow` - Select the first Invoke AWS Lambda Function block and change it to the GetLambdasFunction that you enabled previously (in step 2). In the same block, set the value for the `environment` key to the same environment as the CloudFormation stack.
+  ![invoke_lambda](Images/invoke_lambda_block.png)
+7. Open your `Default customer queue` contact flow and edit the Loop Prompt block. Remove the default queue hold message. Keeping this message will create is disjointed caller experience for non-english calls.
```

Comment:
will create "a" disjointed, instead of "is"?

Reply:
Also, capitalize English

Reply:
fixed